### PR TITLE
Es6 projects common modules article rich links

### PR DESCRIPTION
**Pull request by regiskuckaertz**

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/article/rich-links.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+import config from 'lib/config';
+import { isBreakpoint, getBreakpoint } from 'lib/detect';
+import fetchJson from 'lib/fetch-json';
+import mediator from 'lib/mediator';
+import reportError from 'lib/report-error';
+import template from 'lodash/utilities/template';
+import { spaceFiller } from 'common/modules/article/space-filler';
+import richLinkTagTmpl from 'raw-loader!common/views/content/richLinkTag.html';
+
+const richLinkTag = template(richLinkTagTmpl);
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/article/rich-links.js]: Would you be interested in inlining the template and use an ES6 template literal instead? It's the only placed where it's used ...

**Review by gu-stav — APPROVED** [review]

File: static/src/javascripts/projects/common/modules/article/rich-links.js [hunk 2]
```diff
@@ -8,11 +8,16 @@ import { isBreakpoint, getBreakpoint } from 'lib/detect';
...
-import richLinkTagTmpl from 'raw-loader!common/views/content/richLinkTag.html';
 
-const richLinkTag = template(richLinkTagTmpl);
+const richLinkTag = ({ href }: { href: string }): string => 
+    `<aside class=" element element-rich-link element-rich-link--tag 
+                    element--thumbnail element-rich-link--not-upgraded" 
+            data-component="rich-link-tag" 
+            data-link-name="rich-link-tag"
+            >
+        <p><a href="<%=href%>"><%=href%></a></p>
```
Comment (gustavpursche) [inline static/src/javascripts/projects/common/modules/article/rich-links.js]: `<p><a href="<%=href%>"><%=href%></a></p>` -> `<p><a href="${href}">${href}</a></p>`

Comment (regiskuckaertz) [reply static/src/javascripts/projects/common/modules/article/rich-links.js]: 😆  yeah

**Review by SiAdcock — APPROVED** [review]
👏

**Pull request merged**